src/main/java/mekanism/common/tile/machine/TileEntityChemicalOxidizer.java

```diff
@@ -51,50 +62,90 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-public class TileEntityChemicalOxidizer extends TileEntityProgressMachine<ItemStackToGasRecipe> implements ItemRecipeLookupHandler<ItemStackToGasRecipe> {
+public class TileEntityChemicalOxidizer extends TileEntityProgressMachine<ChemicalOxidizerRecipe> implements ItemRecipeLookupHandler<ChemicalOxidizerRecipe> {
 
     private static final List<RecipeError> TRACKED_ERROR_TYPES = List.of(
           RecipeError.NOT_ENOUGH_ENERGY,
           RecipeError.NOT_ENOUGH_INPUT,
           RecipeError.NOT_ENOUGH_OUTPUT_SPACE,
           RecipeError.INPUT_DOESNT_PRODUCE_OUTPUT
     );
-    public static final long MAX_GAS = 10 * FluidType.BUCKET_VOLUME;
+    public static final long MAX_CHEMICAL = 10 * FluidType.BUCKET_VOLUME;
     public static final int BASE_TICKS_REQUIRED = 5 * SharedConstants.TICKS_PER_SECOND;
 
-    @WrappingComputerMethod(wrapper = ComputerChemicalTankWrapper.class, methodNames = {"getOutput", "getOutputCapacity", "getOutputNeeded",
-                                                                                        "getOutputFilledPercentage"}, docPlaceholder = "output tank")
-    public IGasTank gasTank;
+    //@WrappingComputerMethod(wrapper = ComputerChemicalTankWrapper.class, methodNames = {"getOutput", "getOutputCapacity", "getOutputNeeded",
```

Comment:
this needs to be fixed

Reply:
My mistake.  I'll clean this up.  Copy/paste issue that I solved by commenting, but I forgot to remove it.